cmd/tail.go

```diff
@@ -120,8 +121,19 @@ func tailMain(cmd *cobra.Command, args []string) exitCode {
 	}
 
 	if !follow {
-		cmdutil.ErrPrintf("Please use -f, other operations are not yet supported\n")
-		return exitCode{1}
+		// Defer to `/usr/bin/tail`
```

Comment:
Should this be documented somewhere?

Reply:
Yeah, definitely. I'll add it to the long help.